sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -924,6 +925,61 @@ spec:
 When you save the changes and leave the editor, your changes will be applied to
 the project.
 
+### Changing Github status notifications
+
+By default, Semaphore will create status checks only for initial pipeline.
+
+To modify this, edit the status property in repository section in the project
+YAML.
+
+For example, to modify the granularity of statuses from pipeline to block level
```

Comment:
I don't understand what block-level means with the YML below. What's different when you set level to block?

My guess is that, for example, if your `semaphore.yml` defines two sequential blocks A and B — then you could say "I want my GH status to be just based on block A". Is that the case?

Reply:
I rewrite this part, not it should be more clear. Thanks for feedback.